__tests__/JudoPayTest.tsx

```diff
@@ -0,0 +1,13 @@
+// @flow
+import 'react-native'
+import React from 'react'
+import JudoPay from '../JudoPay';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
+
+test('JudoPay after init has isSandboxed = true', () => {
+  const judoPay = new JudoPay("ta", "atsh")
+  expect(judoPay.isSandboxed).toBe(true);
```

Comment:
Won't this fail though?
`isSandboxed` isn't set to `true` by default. Or is this done on purpose to showcase a failed scenario?

Reply:
As I can see isSandboxed is set true right after var init.

Reply:
Maybe it's a bug, so we should make it false by default

Reply:
![image](https://user-images.githubusercontent.com/53521805/79449007-1aafea80-7feb-11ea-81f5-fa06133ad2f5.png)

Reply:
We set it to `true` in the sample app

Reply:
![Screenshot 2020-04-16 14 05 52](https://user-images.githubusercontent.com/36472137/79449157-6367a380-7feb-11ea-8f23-d0306c6bd112.png)

Reply:
😮 
Ok, my mistake. All good 👍